test/onnx/test_fx_to_onnx.py

```diff
@@ -242,20 +242,6 @@ def forward(self, input):
                 export_options=torch.onnx.ExportOptions(onnx_registry=registry),
             )
 
-        try:
-            torch.onnx.dynamo_export(
-                TraceModel(),
-                x,
-                export_options=torch.onnx.ExportOptions(onnx_registry=registry),
-            )
-        except torch.onnx.OnnxExporterError as e:
-            assert_has_diagnostics(
```

Comment:
Maybe this should stay until sarif is deleted?

Reply:
I removed because the OnnxExporterError contains the information being checked here.